setup.py

```diff
@@ -115,6 +115,7 @@
     "psutil",
     "pydantic",
     "pytest",
+    "pytest-sugar",
```

Comment:
This addition makes it much easier to monitor how the slow tests are going. Beforehand, we had no way of knowing at which % the slow tests were, now if one opens the run it will show how much of the suite has ran.

Reply:
you might want to add this one too:
https://huggingface.co/transformers/testing.html#instantly-shows-failed-tests

I used to use `pytest-sugar` for a while but then I remember there was some problem that I removed it but I can't recall now which.